src/inmanta/server/services/compilerservice.py

```diff
@@ -638,7 +638,17 @@ def _calculate_recompile_wait(
         return wait
 
     async def _auto_recompile_wait(self, compile: data.Compile) -> None:
-        wait_time = opt.server_autrecompile_wait.get()
+        auto_recompile_wait = opt.server_autrecompile_wait.get()
```

Comment:
I think we should invoke the `is_set()` method here: https://github.com/inmanta/inmanta-core/blob/6b07442415f8a706c5ce3310cb5e80ea714fdbca/src/inmanta/config.py#L148